.gitignore

```diff
@@ -4,7 +4,7 @@ out/
 .vscode/*
 !.vscode/launch.json
 !.vscode/tasks.json
-scrypts/
+artifacts/
```

Comment:
do same thing for other ignore file ,like: [.prettierignore](https://github.com/sCrypt-Inc/scryptTS-examples/blob/master/.prettierignore)